README.md

```diff
@@ -37,19 +37,74 @@ git clone https://github.com/PetJournal/petjournal.api.git
 ```
 
-4. Garanta que o [**Node.js**](https://nodejs.org/en/download/) está instalado em sua máquina e então habilite o gerenciador de pacotes yarn usando o comando `corepack enable`;
+4. Garanta que o [**Node.js**](https://nodejs.org/en/download/) está instalado em sua máquina e então habilite o gerenciador de pacotes `yarn` usando o comando `corepack enable`;
 
 5. Acesse a pasta do projeto com seu terminal;
 
 6. Rode o comando `yarn` para instalar as dependências do projeto;
 
-7. Após a correta configuração acima, rode a aplicação com o comando `yarn dev`. Você deve receber a seguinte mensagem de confirmação:
+7. Faça as migrações no banco de dados;
+```bash
+npx prisma migrate dev
+```
 
+8. Rode o comando `yarn build` para transpilar o código typescript para javascript;
+
+9. Rode a aplicação com o comando `yarn start`. Você deve receber a seguinte mensagem de confirmação:
 ```bash
-Server running at http://localhost:<porta>
+Server running at http://localhost:<port>
 ```
 
-8. Para visualizar a documentação com as rotas disponíveis, acesse `localhost:<porta>/api/docs`.
+10. Para visualizar a documentação com as rotas disponíveis, acesse `localhost:<port>/api/docs`.
+
+## Variáveis de ambiente
+
+⚠️ É imprescindível que todas as variáveis estejam devidamente configuradas. ⚠️
+
+`NODE_ENV`: Essa variável define o ambiente que a aplicação está sendo executada. Exemplo: `NODE_ENV=development`.
+
+---
+Variáveis relacionadas as configurações do banco de dados. Como exemplo são os valores padrões para o desenvolvimento.
+
+`POSTGRES_USER`: Usuário do Postgres. Exemplo: `POSTGRES_USER=admin`.
+
+`POSTGRES_PASSWORD`: Senha do Postgres. Exemplo: `POSTGRES_PASSWORD=admin`.
+
+`POSTGRES_DB`: Nome do banco de dados. Exemplo: `POSTGRES_DB=database`.
+
+`PG_HOST`: Endereço do BD. Exemplo: `PG_HOST=localhost`.
+
+`PG_PORT`: Porta do BD. Exemplo: `PG_PORT=54320`.
+
+`DATABASE_URL`: Url fornecida pelos provedores. Exemplo `DATABASE_URL="postgresql://admin:admin@localhost:54320/database?schema=public"`
+
+---
+Variáveis relacionadas as configurações de serviços externos.
+
+`PORT`: Porta da aplicação. Exemplo: `PORT=3333`.
+
+`SALT`: Valor relacionado ao algoritmo de encriptação do Bcrypt. Exemplo: `SALT=12`.
+
+`SECRET`: Chave secreta para a criação do token jwt. Exemplo: `SECRET=supersecretkey`.
+
+`EXPIRY_TIME_SECONDS`: Tempo de expiração em segundos para o código de recuperação de senha. Exemplo `EXPIRY_TIME_SECONDS=3000`.
+
+`MAIL_SERVICE`: Provedor do serviço de email, relacionada à biblioteca de envio de email. Exemplo: `MAIL_SERVICE="gmail"`.
+
+`MAIL_USER`: Email utilizado pela biblioteca. Exemplo: `MAIL_USER="mail_user@mail.com"`.
+
+`MAIL_PASS`: *Credenciais fornecidas pelo provedor de email. Exemplo: `MAIL_PASS="mail_pass"`.
+
+---
+
+\* Exemplo de como obter o `MAIL_PASS` no provedor de email Gmail:
+1. Clica na sua foto no canto e vá em "Gerenciar sua conta do Google".
+2. Na esquerda clique em "segurança".
```

Comment:
could put "Entre nas configurações de segurança da sua conta do Gmail" in place of step 1 and 2